test/e2e/bpmn.rendering.test.ts

```diff
@@ -16,7 +16,7 @@
 import { findFiles } from '../helpers/file-helper';
 import { BpmnDiagramPreparation, BpmnLoadMethod, ImageSnapshotConfigurator, ImageSnapshotThresholdConfig, PageTester } from './helpers/visu-utils';
 
-describe('no visual regression', () => {
+describe('no BPMN elements visual regression', () => {
```

Comment:
:warning: introduce too much changes not related to the purpose of this PR. It is quite hard to see what is changed here when dozen of files are touched

Reply:
ok